lib/nfc/protocols/mf_classic/mf_classic.h

```diff
@@ -232,6 +240,48 @@ bool mf_classic_is_allowed_access(
     MfClassicKeyType key_type,
     MfClassicAction action);
 
+inline __attribute__((always_inline)) MfClassicDict*
+    mf_classic_dict_alloc(const char* path, MfClassicDictMode mode) {
+    return (MfClassicDict*)keys_dict_alloc(path, (KeysDictMode)mode, MF_CLASSIC_KEY_SIZE);
+}
```

Comment:
Please, delete these mf_classic_dict* functions here and below. It's totally OK if nfc app will use KeysDict directly.